interface/forms/care_plan/new.php

```diff
@@ -148,99 +166,111 @@ function set_related(codetype, code, selector, codedesc) {
+                                <textarea name="description[]"  id="description_1" class="form-control description"  rows="3" ><?php echo text($obj{"description"}); ?></textarea>
+                            </div>
+                            <div class="forms col-xs-2">
+                                <label for="code_date_1" class="h5"><?php echo xlt('Date'); ?>:</label>
+                                <input type='text' id="code_date_1"  name='code_date[]' class="form-control code_date datepicker" <?php echo attr($disabled) ?> value='<?php echo attr($obj{"date"}); ?>' title='<?php echo xla('yyyy-mm-dd Date of service'); ?>' />
+                            </div>
+                            <div class="forms col-xs-2">
+                                <label for="care_plan_type_1" class="h5"><?php echo xlt('Type'); ?>:</label>
+                                <select name="care_plan_type[]" id="care_plan_type_1" class="form-control care_plan_type">
+                                    <option value=""></option>
+                                    <?php foreach ($care_plan_type as $value) :
+                                        $selected = ($value['value'] == $obj{"care_plan_type"}) ? 'selected="selected"' : '';
+                                    ?>
+                                    <option value="<?php echo attr($value['value']);?>" <?php echo $selected;?>><?php echo text($value['title']);?></option>
+                                    <?php endforeach;?>
+                                </select>
+                            </div>
+                            <div class="forms col-xs-1 " style="padding-top:35px">
+                                <i class="fa fa-plus-circle fa-2x" aria-hidden="true" onclick="duplicateRow(this.parentElement.parentElement.parentElement);" title='<?php echo xla('Click here to duplicate the row'); ?>'></i>
+                                <i class="fa fa-times-circle fa-2x text-danger"  aria-hidden="true" onclick="deleteRow(this.parentElement.parentElement.parentElement.id);"  title='<?php echo xla('Click here to delete the row'); ?>'></i>
+                            </div>
+                            <div class="clearfix"></div>
+                        </div>
+                    </div>
+
+                    <?php }
+                    ?>
+
+                </fieldset>
+                <div class="form-group">
+                    <div class="col-sm-12 text-center">
+                        <div class="btn-group" role="group">
+                            <a href="javascript:top.restoreSession();document.my_form.submit();" class="btn btn-default btn-save"><?php echo xlt('Save'); ?></a>
+                            <a href='<?php echo "$rootdir/patient_file/encounter/$returnurl";?>' class="btn btn-default btn-cancel" onclick="top.restoreSession()"><?php echo xlt('Don\'t Save'); ?></a>
```

Comment:
@robertdown ,
The btn-link does not seem to look very good on zbig01's theme. Rec testing and seeing what I mean on the other forms where btn-link is used. I think this may be because zbig01's buttons are colored(ie. more obvious than the buttons in light theme). Is there a way for a theme to override this btn-link styling for the de-emphasized buttons?